kubernetes/provider.go

```diff
@@ -230,7 +237,12 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		return nil, fmt.Errorf("Failed to configure: %s", err)
 	}
 
-	return k, nil
+	a, err := aggregator.NewForConfig(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to configure: %s", err)
+	}
+
+	return &KubeClientsets{k, a}, nil
```

Comment:
The new aggregator client set is required as `apiregistration` API is not part of the main one.

This felt like the proper way to initialize both client sets and be able to retrieve them from resource functions.